### PR TITLE
Spec `?animated` on `/thumbnail`

### PR DESCRIPTION
**Pull request by turt2live**
Better with https://github.com/matrix-org/matrix-spec/pull/1756

See https://github.com/matrix-org/matrix-spec-proposals/pull/2705

Preview: https://pr1757--matrix-spec-previews.netlify.app

### CODE REVIEW

File: data/api/client-server/content-repo.yaml [hunk 1]
```diff
@@ -633,6 +633,28 @@ paths:
           schema:
             type: boolean
             default: false
+        - in: query
+          name: animated
+          x-addedInMatrixVersion: "1.10"
```
Comment (richvdh) [inline data/api/client-server/content-repo.yaml:638]:
```suggestion
          x-addedInMatrixVersion: "1.11"
```

**Review by richvdh — COMMENTED** [review]
Could you merge the latest changes from `main` so we can see how this looks in the preview?

File: data/api/client-server/content-repo.yaml (hunk 1 above)
Comment (turt2live) [reply data/api/client-server/content-repo.yaml]: applied manually - thanks!

File: data/api/client-server/content-repo.yaml [hunk 2]
```diff
@@ -644,14 +666,33 @@ paths:
...
           content:
             image/jpeg:
               schema:
                 # This is a workaround for us not being able to say the response is required.
                 description: "**Required.** The bytes for the thumbnail."
             image/png:
               schema:
-                description: "**Required.** The bytes for the thumbnail."
+                x-changedInMatrixVersion:
+                  "1.11": The PNG may be of the APNG variety if animation is supported and requested.
```
Comment (richvdh) [inline data/api/client-server/content-repo.yaml:680]: For the record (no action required): these don't seem to show up in the rendered spec, though that's no reason not to add them.

**Review by richvdh — APPROVED** [review]

File: data/api/client-server/content-repo.yaml (hunk 2 above)
Comment (turt2live) [reply data/api/client-server/content-repo.yaml:680]: https://github.com/matrix-org/matrix-spec/issues/1774

**Pull request merged**